srml/support/src/traits.rs

```diff
@@ -153,7 +153,7 @@ impl<Imbalance> OnUnbalanced<Imbalance> for () {
 }
 
 /// Simple boolean for whether an account needs to be kept in existence.
-#[derive(Copy, Clone, Eq, PartialEq)]
+#[derive(Copy, Clone, Eq, PartialEq, Encode, Decode, Debug)]
```

Comment:
```suggestion
#[derive(Copy, Clone, Eq, PartialEq, Encode, Decode)]
#[cfg_attr(feature = "std", derive(Debug))]
```

Debug only on std, or use the new `RuntimeDebug` (which I am not 100% sure how it works but you can find examples probably cc @tomusdrw )

Reply:
Indeed, please use `RuntimeDebug`. It's just regular `derive`.